notifique/src/main/java/com/nathanrassi/notifique/NotifiqueListView.kt

```diff
@@ -77,13 +79,15 @@ internal class NotifiqueListView(
     fun onSelectionStateChanged(selected: Boolean)
   }
 
-  fun deleteSelected() {
+  fun deleteSelected(bool: Boolean) {
```

Comment:
this is confusing. use a different api. `fun deselectSelected` separate from deleteSelected.
```kotlin
fun deleteSelected() {
  selectionTracker.clearSelection()
}
```